aaelf32/aaelf32.rst

```diff
@@ -0,0 +1,3289 @@
+  | PT_ARM_ARCHEXT_FMTMSK  | 0xff000000 | Masks bits describing the format of data in subsequent words. The masked value is           |
+  |                        |            | described in `Architecture compatibility data formats`_, below.                             |
+  +------------------------+------------+---------------------------------------------------------------------------------------------+
+  | PT_ARM_ARCHEXT_PROFMSK | 0x00ff0000 | Masks bits describing the architecture profile required by the executable. The masked value |
+  |                        |            | is described in `Architecture profile compatibility data`_, below.                          |
+  +------------------------+------------+---------------------------------------------------------------------------------------------+
+  | PT_ARM_ARCHEXT_ARCHMSK | 0x000000ff | Masks bits describing the base architecture required by the executable. The masked value is |
+  |                        |            | described in `Architecture version compatibility data`_, below.                             |
+  +------------------------+------------+---------------------------------------------------------------------------------------------+
+
+The following table lists the architecture compatibility data formats defined
+by this ABI. All other format identifiers are reserved to future revisions of
+this specification.
+
+.. _Architecture compatibility data formats:
+
+.. table:: Architecture compatibility data formats
+  :widths: 35 15 50
+
+  +------------------------+------------+----------------------------------------------------------------------------------------+
+  | Name                   | Value      | Meaning                                                                                |
+  +========================+============+========================================================================================+
+  | PT_ARM_ARCHEXT_FMT_OS  | 0x00000000 | There are no additional words of data.  However, if EF_OSABI is non-zero, the relevant |
+  |                        |            | platform ABI may define additional data that follows the initial word.                 |
+  +------------------------+------------+----------------------------------------------------------------------------------------+
+  | PT_ARM_ARCHEXT_FMT_ABI | 0x01000000 | `Platform architecture compatibility data (ABI format)`_, below, describes the format  |
+  |                        |            | of the following data words.                                                           |
+  +------------------------+------------+----------------------------------------------------------------------------------------+
+
+The following table lists the values specifying the architectural profile
+needed by an executable file.
+
+.. _Architecture profile compatibility data:
+
+.. table:: Architecture profile compatibility data
+  :widths: 40 15 45
```

Comment:
as described above on the version history table, please use a class to specify the width